src/stream/overlayText.js

```diff
@@ -34,8 +36,9 @@ const getOverlayTextString = async (path, config, typeKey, metadata) => {
     // Check if we have an artist option
     if (overlayConfigObject.artist && overlayConfigObject.artist.enabled) {
       const itemObject = overlayConfigObject.artist;
+      const safeText = safeStrings.forFilter( itemObject.label .  metadata.common.artist);
```

Comment:
So I tested this for you 😄 

And I got the following error:

<img width="713" alt="screen shot 2018-10-09 at 1 48 09 am" src="https://user-images.githubusercontent.com/1448289/46657710-c6aa8700-cb65-11e8-8f72-e11079b3b850.png">

I noticed you did `const safeText = safeStrings.forFilter(itemObject.label . metadata.common.artist);`

I think that is PHP syntax? 😂 

Thanks again for being so responsive!

Reply:
You're absolutely right! I was working on a PHP project before and JS syntax is so similar I didn't even notice!